modules/ai/assets/js/gutenberg/index.js

```diff
@@ -93,6 +84,21 @@ import { EditTextWithAi } from './edit-text-with-ai';
 			addAiIndicator( 'featured-image', addGenerateFeaturedImageWithAI );
 			addAiIndicatorToTextBlock( [ 'paragraph', 'heading' ] );
 		} );
+
+		const observer = new MutationObserver( ( mutationsList ) => {
+			for ( const mutation of mutationsList ) {
+				if ( 'childList' === mutation.type ) {
+					if ( document.querySelector( '.editor-post-excerpt' ) ) {
+						addGenerateExcerptWithAI();
+					}
+				}
+			}
+		} );
+
+		observer.observe( document.body, { childList: true, subtree: true } );
```

Comment:
Maybe observe only the sidebar instead of all changes in the body

Reply:
it's being added to the body - not to the sidebar